src/AppServices/KafkaPage/KafkaPage.tsx

```diff
@@ -35,50 +49,212 @@ export const KafkaPage: FunctionComponent = () => {
+                justifyContent={{ default: "justifyContentCenter" }}
+                flex={{ default: "flex_1" }}
+              >
+                <FlexItem>
+                  <DescriptionList isHorizontal>
+                    <DescriptionListGroup>
+                      <DescriptionListTerm>
+                        {t("steamingUnit")}
+                      </DescriptionListTerm>
+                      <DescriptionListDescription>
+                        {"$1.49/hour"}
+                      </DescriptionListDescription>
+                    </DescriptionListGroup>
+                    <DescriptionListGroup>
+                      <DescriptionListTerm>
+                        {t("dataTransfer")}
+                      </DescriptionListTerm>
+                      <DescriptionListDescription>
+                        {"$0.09/GB"}
+                      </DescriptionListDescription>
+                    </DescriptionListGroup>
+                    <DescriptionListGroup>
+                      <DescriptionListTerm>{t("storage")}</DescriptionListTerm>
+                      <DescriptionListDescription>
+                        {"$0.10/GB"}
+                      </DescriptionListDescription>
+                    </DescriptionListGroup>
+                  </DescriptionList>
+                </FlexItem>
+              </Flex>
+              <Divider isVertical />
+              <FlexItem flex={{ default: "flex_1" }}>
+                <TextContent>
+                  <Text component={TextVariants.p}>
+                    <Trans
+                      i18nKey={"kafkaoverview:streamingUnitText"}
+                      values={{
+                        value: "streaming unit",
+                      }}
+                    />
+                  </Text>
+                </TextContent>
+              </FlexItem>
+            </Flex>
+          </CardBody>
+        </Card>
+      </MarketingPageSection>
+      <MarketingPageSection kafkaOverviewPage={true}>
+        <Card>
+          <CardTitle>
+            <Title headingLevel="h2">{t("kafkaInstanceCapacityTitle")}</Title>
+          </CardTitle>
+          <CardBody>
+            <Stack hasGutter>
+              <Text component={TextVariants.p}>
+                {t("kafkaInstanceCapacityDescription")}
+              </Text>
+              <KafkaInstanceCapacityTable />
+            </Stack>
+          </CardBody>
+          <CardFooter>
+            <TextContent className={"pf-u-font-size-sm"}>
+              {t("kafkaInstanceCapacityfooter1")}{" "}
+              <Button
+                data-testid="redhatSupport-buttonCTA"
+                isInline
+                variant={ButtonVariant.link}
+                component="a"
+                target="_blank"
+                href="https://access.redhat.com/support"
+              >
+                {" "}
+                Redhat support
+                <ExternalLinkAltIcon className="pf-u-ml-xs" />
```

Comment:
Even though this icon is less likely to wrap to a new line, it would be good to add a non-breaking space `&nbsp;` here as well.

Something to look for is whether line breaks in code will result in extra whitespace in html. If there is any whitespace before or after the `&nbsp;` then it will still result in line breaks.